daemon/api_quotas.go

```diff
@@ -182,6 +182,7 @@ func getQuotaGroupInfo(c *Command, r *http.Request, _ *auth.UserState) Response
 		GroupName:   group.Name,
 		Parent:      group.ParentGroup,
 		Snaps:       group.Snaps,
+		Services:    group.Services,
```

Comment:
does this change need a unit test?

Reply:
Really good point, I will address it in the same followup

Reply:
Yes great point! This has been addressed.